engine/src/org/pentaho/di/trans/steps/databaselookup/DatabaseLookup.java

```diff
@@ -215,7 +214,7 @@ public DatabaseLookup( StepMeta stepMeta, StepDataInterface stepDataInterface, i
     return outputRow;
   }
 
-  private void storeRowInCache( RowMetaInterface lookupMeta, Object[] lookupRow, Object[] add ) {
+  void storeRowInCache( RowMetaInterface lookupMeta, Object[] lookupRow, Object[] add ) {
```

Comment:
If these are package-private for a reason, please add Javadoc and/or annotations. If it is to expose this method for testing, we've been using Guava's @VisibleForTesting annotation.